Terminal.Gui/Views/ListView.cs

```diff
@@ -307,12 +307,12 @@ public override void Redraw (Rect bounds)
 		/// <summary>
 		/// This event is raised when the selected item in the <see cref="ListView"/> has changed.
 		/// </summary>
-		public event EventHandler<ListViewItemEventArgs> SelectedChanged;
+		public Action<ListViewItemEventArgs> SelectedChanged;
```

Comment:
`SelectedItemChanged`

Reply:
But wouldn't that break with code already made by users?